cloudify_gcp/admin/projects.py

```diff
@@ -24,10 +26,18 @@
 
 class Project(CloudResourcesBase):
 
-    def __init__(self, config, logger, project_id=None, name=None):
-        super(CloudResourcesBase, self).__init__(config, logger)
-        self.project_id = utils.get_gcp_resource_name(project_id)
-        self.name = name if name else self.project_id
+    def __init__(self,
+                 config,
+                 logger,
+                 name=None,
+                 project_id=None,
+                 parent=None):
+
+        super(Project, self).__init__(config, logger)
+
+        self.project_id = project_id if project_id else name
+        self.name = name if name else project_id
+        self.parent = parent
 
```

Comment:
It could be a little prettier to move the project body payload to a property. Then the code below gets a little cleaner.
```
@property
def project_body(self):
    project_body = {
        'name': self.name,
        'projectId': self.project_id,
        'parent': self.parent
    }
    self.logger.debug('Project info: {}'.format(repr(project_body)))
    return project_body
```